content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,45 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The Control Plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside a group of containers called
+a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. The idea of a pod
+is to isolate different workloads from each other. The computers
+that run pods are called _nodes_, and one node can run pods from many
+different workloads.
+
+To make this happen, Kubernetes runs several components on each node:
+
+### kubelet
+
+{{< glossary_definition term_id="kubelet" length="all" >}}
+
+### kube-proxy
+
+{{< glossary_definition term_id="kube-proxy" length="all" >}}
+
+### Container runtime
+
+{{< glossary_definition term_id="container-runtime" length="all" >}}
+
+## Control plane components
```

Comment:
https://www.bodden.de/2008/07/16/english-is-right-associative-french-is-not/

English is right-associative. To group a pair of words that together act as an adjective, they should be hyphenated. So: "Control-plane components". 

One exception to this rule is a group of words acting as a proper noun. So, if "Control Plane" is being used in that sense, then Plane should be capitalized, as it was before, and as it appears in the following paragraph.

Reply:
I used Google to look for common practice. Most pages on the web that match a Google search for either “control plane components” or “control-plane components” use the former.

Let's retain the current wording @bgrant0607.